### DISCUSSION TITLE
New empty turbo repo

### DISCUSSION THREAD

**Discussion by wmtrinu** [vercel/turborepo#7934] [Help]
### Summary

Hey

Is there a way to create an empty new project with turbo? I've checked the documents but could not find a thing which is a bit frustrating. (new to turborepo...)

I'd like to create apps and shared packages (incrementally)

`apps`
- core service (nest js)
- auth service (nest js)
- web (next js application)

`shared`
- ui library
- eslint configuration
- typescript configuration
- prisma

**Comment by anthonyshew** [comment]
Would this page maybe help? https://turbo.build/repo/docs/getting-started/create-new

If not, I'm not sure I understand the question.
